.github/workflows/pt-nextjs-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Check NextJs build
+
+on:
+  pull_request:
+    types: [opened, reopened]
+
+  workflow_dispatch:
+
+env:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Detect package manager
+        id: detect-package-manager
+        run: |
+          if [ -f "${{ github.workspace }}/yarn.lock" ]; then
+            echo "manager=yarn" >> $GITHUB_OUTPUT
+            echo "command=install" >> $GITHUB_OUTPUT
+            echo "runner=yarn" >> $GITHUB_OUTPUT
+            exit 0
+          elif [ -f "${{ github.workspace }}/package.json" ]; then
+            echo "manager=npm" >> $GITHUB_OUTPUT
+            echo "command=ci" >> $GITHUB_OUTPUT
+            echo "runner=npx --no-install" >> $GITHUB_OUTPUT
```

Comment:
Why runner if you never use it?

Reply:
Used as cache key `${{ runner.os }}`

Reply:
But in line 46 you didnt use with ${{ steps.detect-package-manager.outputs.runner }}, why?

Reply:
But in line 46 you didnt use with ${{ steps.detect-package-manager.outputs.runner }}, why?

Reply:
@Wabri  runner is an alias, not a step prop

Reply:
I have to agree with @Wabri. You are creating a new output variables, but at the moment you are not actually using it like the others.

It's not a huge mess as I can say, it can become handy in case we want to do stuff like:

`${{ steps.detect-package-manager.outputs.runner }} next build `

I think it can be up to you to keep it or delete it, I do not have strong opinions about it.